1-mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,4 +1,20 @@
-function setAlarm() {}
+function setAlarm() {
+  let timeRemaining = document.getElementById("timeRemaining");
+  let input = document.getElementById("alarmSet");
+  let inputValue = input.value;
+  countdown = setInterval(() => {
+    let minutes = Math.floor(inputValue / 60);  
+    let seconds = inputValue % 60;
+    timeRemaining.innerText = `Time remaining: ${minutes}:${seconds}`;
+    if (inputValue === 0) {
```

Comment:
`0` magic number. Lets make this: `const ZERO = 0` so we can share it across functions, a name that describes what this `0` is used for would be helpful, for example: `const TIME_ELAPSED=0`